### PR TITLE
docs(nxdev): rewrite getting started section

### PR DESCRIPTION
**Pull request by isaacplmann**
Rewrites the getting started section

Getting Started
- Intro to Nx (Getting Started renamed)
- Nx Setup (Folder Structure content is moved into here)
- Nx CLI (CLI Overview and Nx and CLI content is combined here)
- Nx Editor Plugins (Renamed from Nx Console page)

Getting Started/Configuration is moved to Core Concepts.

Resources page is deleted. (Should we move this somewhere else instead?)

### CODE REVIEW

File: docs/angular/getting-started/nx-setup.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+`/libs/` contains the library projects. There are many different kinds of libraries, and each library defines its own external API so that boundaries between libraries remain clear.
+
+`/tools/` contains scripts that act on your code base. This could be database scripts, [custom executors](/{{framework}}/executors/creating-custom-builders) (or builders), or [workspace generators](/{{framework}}/generators/workspace-generators).
+
+`/workspace.json` defines each project in your workspace and the executors that can be run on those projects.
+
+`/nx.json` adds extra information about projects, including manually defined dependencies and tags that can be used to restrict the ways projects are allowed to depend on each other.
+
+`/tsconfig.json` sets up the global TypeScript settings and creates aliases for each library to aid when creating TypeScript imports.
```
Comment (vsavkin) [inline docs/angular/getting-started/nx-setup.md:71]: this is tsconfig.base.json

File: docs/angular/getting-started/nx-setup.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+```treeview
+myorg/
+├── apps/
+├── libs/
+├── tools/
+├── workspace.json
+├── nx.json
+├── package.json
+└── tsconfig.json
```
Comment (vsavkin) [inline docs/angular/getting-started/nx-setup.md:56]: tsconfig.base.json

**Review by caitlinthefirst — CHANGES_REQUESTED** [review]
My edits/suggestions are here: https://docs.google.com/document/d/1c0vD1TzdicGNbIqNr7oX_ObQ2yjsaWNxnuX_Qhb6wYE/edit?usp=sharing

If I can get permission to make in-line edits, I'm happy to do that as well.

**Comment by isaacplmann** [comment]
Included @brandonroberts 's edits. And all of @caitlinthefirst 's edits, except the comment about referencing Nx Cloud.

**Review by brandonroberts — APPROVED** [review]

**Pull request merged**